quartz_solar_forecast/data.py

```diff
@@ -181,7 +180,7 @@ def process_pv_data(live_generation_kw: pd.DataFrame, ts: pd.Timestamp, site: PV
 
 def make_pv_data(site: PVSite, ts: pd.Timestamp) -> xr.Dataset:
     """
-    Make PV data by combining live data from SolarEdge or Enphase and fake PV data.
+    Make PV data by combining live data from SolarEdge, Enphase, or Solis and fake PV data.
```

Comment:
I think if we are removing the SolarEdge stuff we should remove all mentions of it for now like in this comment